src/lib/hooks/useBuyNowPrice.ts

```diff
@@ -0,0 +1,55 @@
+//- React Imports
+import { useQuery } from 'react-query';
+
+//- Hooks Imports
+import { useZnsSdk } from './useZnsSdk';
+
+export interface UseBuyNowPriceReturn {
+	buyNowPrice: string;
+	error: any;
+	isError: boolean;
+	isLoading: boolean;
+	isSuccess: boolean;
+}
+
+export const useBuyNowPrice = (domainId: string): UseBuyNowPriceReturn => {
+	// SDK
+	const sdk = useZnsSdk();
+
+	// Query
+	const {
+		error,
+		isError,
+		isLoading,
+		isSuccess,
+		data: { buyNowPrice } = {},
+	} = useQuery(
+		`domain-buy-now-price-${domainId}`,
+		async () => {
+			try {
```

Comment:
Gotcha, I was going to ask if we needed to handle errors here. So we'd expect to see the following in the console for undefined ? Are we able to prevent these logging in the console?

<img width="369" alt="Screenshot 2022-08-10 at 12 47 49" src="https://user-images.githubusercontent.com/39112648/183893620-1b34cfcf-e9af-4ba2-85e3-f4b75255306c.png">

Reply:
Hm think this needs to be handled in sdk

<img width="605" alt="Screenshot 2022-08-11 at 15 41 44" src="https://user-images.githubusercontent.com/39112648/184160114-f919dac4-7bd7-410a-a0c6-6584af422ea9.png">

Reply:
Correct - this would need to be fixed in the SDK. I'll ping CPT.